Sources/EdgeBridge.swift

```diff
@@ -47,6 +53,32 @@ public class EdgeBridge: NSObject, Extension {
         return true
     }
 
+    private func handleCaptureContextData(event: Event) {
+        // add to existing list
+        contextDataCapturer.addEvent(event)
+    }
+
+    private func handleCaptureRequest(_ event: Event) {
+        var captureState: Bool
+        if let capture = event.data?[EdgeBridgeConstants.EventDataKeys.ContextDataKeys.CAPTURE] as? Bool {
```

Comment:
We can optimize and directly capture the value with guard let.

guard let captureState = event.data?[EdgeBridgeConstants.EventDataKeys.ContextDataKeys.CAPTURE] as? Bool else {
Log.debug(label: EdgeBridgeConstants.LOG_TAG, "Received unknown capture state, ignoring this event.")
return 
}

Reply:
So we will ignore the invalid values. After this we can add logic for captureState true/false as you have.